src/containers/ViewerPanel/index.tsx

```diff
@@ -270,6 +282,8 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
             timeStep,
             receiveTrajectory,
             setBuffering,
+            setIsPlaying,
```

Comment:
Linter says this value is unused and can be deleted

Reply:
Good catch, ty